src/test/java/ay2021s1_cs2103_w16_3/finesse/model/category/CategoryTest.java

```diff
@@ -39,4 +41,35 @@ public void isValidCategoryName() {
         assertTrue(Category.isValidCategoryName("peter*")); // contains non-alphanumeric characters
     }
 
+    @Test
+    public void asciiTest() {
```

Comment:
It may be useful to note here that each character is its own equivalence partition, in case someone casts doubt on the efficiency of this testing later on.